modules/ixBidAdapter.js

```diff
@@ -163,6 +163,27 @@ const MEDIA_TYPES = {
   Native: 4
 };
 
+const AP_VALID_SIZES = [
```

Comment:
Any idea why we have not mentioned all the IAB sizes in here?

`				[120, 600],
				[160, 600],
				[200, 200],
				[240, 400],
				[250, 250],
				[300, 50],
				[300, 100],
				[300, 250],
				[300, 600],
				[320, 50],
				[320, 100],
				[320, 480],
				[336, 280],
				[468, 60],
				[480, 320],
				[580, 400],
				[690, 90],
				[675, 90],
				[670, 90],
				[650, 90],
				[630, 90],
				[600, 90],
				[720, 300],
				[728, 90],
				[728, 250],
				[728, 280],
				[900, 90],
				[970, 90],
				[970, 250]`

Reply:
https://adpushup.slack.com/archives/C06RLQ395SQ/p1711630186060059
Discussed on slack